core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -548,7 +548,7 @@ object DumpLogSegments {
       }
 
       val json = new ObjectNode(JsonNodeFactory.instance)
-      json.set("type", new TextNode(version.toString))
+      json.set("version", new TextNode(version.toString))
```

Comment:
For my understanding, if `type` is changed to `version` here shouldn't it also be changed to `version` in `prepareKey` as well?

Reply:
No, it should not. In the key, we get the type of the record together with the serialized key. In the value, we have the version of the record together with the serialized record. The format is defined here: https://github.com/apache/kafka/blob/trunk/coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRecordSerde.java#L29